### ISSUE TITLE
Tabs Missing Name, Role, and aria-controls Attributes

### ISSUE THREAD

**Issue by Bluefish0558** [microsoft/vscode#227510]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.92.2
- OS Version: 14.6.1 (23G93)

**Description**: Tabs such as "PROBLEMS," "OUTPUT," and "TERMINAL" are missing proper name, role, and aria-controls attributes, making them inaccessible to screen reader users.

**Steps to Reproduce:**

1. Open the tabs "PROBLEMS," "OUTPUT," and "TERMINAL" in VSCode.
2. Use a screen reader to navigate and observe the lack of accessible name, role, and aria-controls attributes.

**User Impact**: Screen reader users are unable to determine that these tabs reveal panels of content or which panel is currently active.

**Expected Behavior**: Tabs should provide proper state and role, including `role="tabpanel"`, `aria-controls` set to the ID of the corresponding tabpanel, and `aria-labelledby` for tabpanel elements.

**Actual Behavior**: Tabs are missing these attributes, reducing accessibility.

**Recommendation**: Ensure each tab provides proper state and role information, including role="tabpanel", aria-controls, and aria-labelledby as needed for accessibility.

Recommended Reading
For more information about creating accessible tab controls, please see the W3C Web Accessibility Initiative's ARIA Authoring Practices Guide: https://www.w3.org/WAI/ARIA/apg/patterns/tabpanel/

<img width="1506" alt="attributes" src="https://github.com/user-attachments/assets/37ca8373-8d9d-4126-bb9a-ee5ff6253028">

**Comment by meganrogge** [comment]
closing as I've had no users reporting this as an issue
